utils/spaceship.detectBrowser/index.js

```diff
@@ -0,0 +1,23 @@
+function detectBrowser() {
+  if (
+    (window.navigator.userAgent.indexOf('Opera') ||
+      window.navigator.userAgent.indexOf('OPR')) != -1
+  ) {
+    return 'Opera';
+  } else if (window.navigator.userAgent.indexOf('Chrome') != -1) {
```

Comment:
How about this?
`!!window.navigator.userAgent.indexOf('Chrome')`

Reply:
👍 Good! It will be good if i change codes to this pattern. thanks.